### PR TITLE
Bookend test refactor

### PR DESCRIPTION
**Pull request by breakpointer**
**Related Issue**  
Supports #ISSUE_NUMBER

**Related PRs**  
This PR is not dependent on any other PR

**What does this PR do?**  
The intent of this section is to help team members understand the scope of your changes and why you are making said changes. If there are any concerns or risks associated with your pull request, they should be documented here so your team members can give informative feedback in their reviews.

**Description of Changes**  
The intent of this section is to describe the implementation details of the changes you made. Any architectural decisions made should be noted here as well as a short description of why you made that decision.

**What gif most accurately describes how I feel towards this PR?**  
Example of a gif

### CODE REVIEW

File: packages/bootleg-tokens/contracts/BookendSharedRoyaltyToken.sol [hunk 1]
```diff
@@ -23,22 +23,32 @@ contract BookendSharedRoyaltyToken is AbstractSharedRoyaltyToken, ERC721Mintable
...
     } 
-    for (uint256 i = start; i < maxCount; i += 1) {
+
+    // Calculating payments for end of bookend
+    for (uint256 i = start; i < maxCount; i++) {
       if (token.franchisors[i - 1] == franchisor) {
-        payment += (token.payments[i] * franchisorPercentage) % 100 == 0 ? (token.payments[i] * franchisorPercentage)/100  : (token.payments[i] * franchisorPercentage)/100 + 1;
+        payment += (token.payments[i] * franchisorPercentage)/100;
+        // check for rounding and round up
+        if ((token.payments[i] * franchisorPercentage) % 100 != 0) {
```
[the remark this answers is not in the document]
Comment (hadasz) [reply packages/bootleg-tokens/contracts/BookendSharedRoyaltyToken.sol]: The round up only happens for _one_ person per payment. The other person in the transaction (the franchisee), has a payment that is _rounded down_. In the end, the sum of the rounded up payment and the rounded down payment equal the total amount paid for the transaction.

File: packages/bootleg-tokens/test/BookendSharedRoyaltyToken.test.js [hunk 2]
```diff
@@ -9,214 +9,148 @@ contract('BookendSharedRoyaltyToken', accounts => {
...
-    expect(parseInt(balance)).toEqual(10);
-  }); //minter -> franchisor -> payment is not integer
 
-  it('count -> minter -> franchisor -> non-integer', async () => {
-    const token = await BookendSharedRoyaltyToken.new(5);
-    await token.mint(accountOne, tokenId);
-    await token.transferFrom(accountOne, accountTwo, tokenId, {
-      value: 7
+    describe('rounding error correction', async () => {
+      it('rounds up the payment amount for accountOne', async () => {
```
[the remark this answers is not in the document]
Comment (hadasz) [reply packages/bootleg-tokens/test/BookendSharedRoyaltyToken.test.js]: Per response to comment above - there aren't multiple accounts in this model of bookend. Just the franchisor and the person who sold.

File: packages/bootleg-tokens/test/BookendSharedRoyaltyToken.test.js [hunk 3]
```diff
@@ -9,214 +9,148 @@ contract('BookendSharedRoyaltyToken', accounts => {
...
-      value: 10
+  describe('the first transfer', async () => {
+    it('pays the accountOne full payment on first transfer', async () => {
+      const token = await BookendSharedRoyaltyToken.new(5);
+      await token.mint(accountOne, tokenId);
+      await token.transferFrom(accountOne, accountTwo, tokenId, {
+        value: 10
+      });
+      const balance = await token.paymentBalanceOf(accountOne, 1, 1, tokenId);
+      expect(parseInt(balance)).toEqual(10);
```
[the remark this answers is not in the document]
Comment (breakpointer) [reply packages/bootleg-tokens/test/BookendSharedRoyaltyToken.test.js]: Yeah, that makes sense. I'll update this to use the helpers.

File: packages/bootleg-tokens/contracts/BookendSharedRoyaltyToken.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (barlock) [reply packages/bootleg-tokens/contracts/BookendSharedRoyaltyToken.sol]: 👍 Forgot that there were only ever two people getting money.

File: packages/bootleg-tokens/test/BookendSharedRoyaltyToken.test.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (breakpointer) [reply packages/bootleg-tokens/test/BookendSharedRoyaltyToken.test.js]: Actually. I'm not going to use the helper. The method, the only method in the BookendSharedRoyaltyToken.sol (currently) is the `paymentBalanceOf` method. Calling a helper here adds indirection that I don't think is helpful. In some test cases we actually want to call `paymentBalanceOf` with specific params.

**Pull request merged**